src/fairseq2/assets/cards/models/wav2vec2.yaml

```diff
@@ -20,6 +20,22 @@ checkpoint: "https://ai.meta.com/pseudo_dinosr_base/;gated=true"
 
 ---
 
+name: wav2vec2_large_ls960
```

Comment:
I don't think "ls960" suffix/prefix is overall necessary. I consider it the default one (since the base is also in fact a pretraining of ls960). So this can be just "wav2vec2_large".

Nit: please move it on top of "dinosr_base" which is a temporary asset that we will eventually remove.

Reply:
I will move it on top of dinosr_base, but I don't think this should be the default one. Simply bcoz the other model is a much better models, and we should have the default model to be the better model. Also, you can see the large model configs to be named as "vox_*.yaml" here, referring to the model trained on LibriVox 60k. I'm happy to call the other one "large" and this one "large_ls960".

Reply:
I don't think the default has to be necessarily to "best" model. It should be just consistent with the literature/paper. According to the last paragraph of section 4.2 (pretraining) in wav2vec2 paper, the base model was solely trained with LS-960. So if we call it just "base", then "large" should imply that it is the same training recipe with just a larger architecture. So we have three variants: base with ls960h, large with ls960h, and large with lv60k. In my opinion "base", "large", and "large_lv60k" sound more consistent.